Kentico/ems-mvc-components
# Added ability to embed Vimeo videos

## Added ability to embed Vimeo videos
### Code changes
- Renamed YoutubeVideo[xxx] files, classes and comments to Video[xxx]
- Added tests for vimeo urls
- Added a new REGEX_VIDEO_URL that checks either youtube or vimeo url formats.
- Added a condition to VideoHelper.GetVideoId to get youtube ids separately to vimeo.
- Added GetVideoEmbedFormat to VideoHelper and replaced the explicit mention of youtube's embed url in the VideoWidget View.

### Motivation

This is adding the ability for the Video widget to render Vimeo videos rather than just Youtube videos.

### Checklist


### How to test

Add a youtube video to the sandbox project, then add a vimeo video to the project. Both should render.
